utils/utils.go

```diff
@@ -125,4 +128,188 @@ func HandlePanic() {
+		return
+	}
+	var keployAlias string
+	getAlias(&keployAlias, logger)
+	keployAlias = keployAlias + cmdName + " -c "
+	var cmd *exec.Cmd
+	if cmdName == "record" {
+		keployAlias = keployAlias + "\"" + recordFlags.Command + "\" "
+		if len(recordFlags.PassThroughPorts) > 0 {
+			portSlice := make([]string, len(recordFlags.PassThroughPorts))
+			for i, port := range recordFlags.PassThroughPorts {
+				portSlice[i] = fmt.Sprintf("%d", port)
+			}
+			joinedPorts := strings.Join(portSlice, ",")
+			keployAlias = keployAlias + " --passThroughPorts=" + fmt.Sprintf("%v ", joinedPorts)
+		}
+		if recordFlags.ConfigPath != "." {
+			keployAlias = keployAlias + " --configPath " + recordFlags.ConfigPath
+		}
+		if len(recordFlags.Path) > 0 {
+			keployAlias = keployAlias + " --path " + recordFlags.Path
+		}
+		addtionalFlags := appendFlags("containerName", recordFlags.ContainerName) + appendFlags("buildDelay ", recordFlags.BuildDelay.String()) + appendFlags("delay", fmt.Sprintf("%d", recordFlags.Delay)) + appendFlags("proxyport", fmt.Sprintf("%d", recordFlags.Proxyport)) + appendFlags("networkName", recordFlags.NetworkName) + appendFlags("enableTele=", fmt.Sprintf("%v", recordFlags.EnableTele))
+		keployAlias = keployAlias + addtionalFlags
+		cmd = exec.Command("sh", "-c", keployAlias)
+
+	} else {
+		keployAlias = keployAlias + "\"" + testFlags.Command + "\" "
+		if len(testFlags.PassThroughPorts) > 0 {
+			portSlice := make([]string, len(recordFlags.PassThroughPorts))
+			for i, port := range recordFlags.PassThroughPorts {
+				portSlice[i] = fmt.Sprintf("%d", port)
+			}
+			joinedPorts := strings.Join(portSlice, ",")
+			keployAlias = keployAlias + " --passThroughPorts=" + fmt.Sprintf("%v ", joinedPorts)
+		}
+		if testFlags.ConfigPath != "." {
+			keployAlias = keployAlias + " --configPath " + testFlags.ConfigPath
+		}
+		if len(testFlags.Testsets) > 0 {
+			testSetSlice := make([]string, len(testFlags.Testsets))
+			for i, testSet := range testFlags.Testsets {
+				testSetSlice[i] = fmt.Sprintf("%v", testSet)
+			}
+			joinedTestSets := strings.Join(testSetSlice, ",")
+			keployAlias = keployAlias + " --testsets=" + fmt.Sprintf("%v", joinedTestSets)
+		}
+		if len(testFlags.Path) > 0 {
+			keployAlias = keployAlias + " --path " + testFlags.Path
+		}
+		addtionalFlags := appendFlags("containerName", testFlags.ContainerName) + appendFlags("buildDelay", testFlags.BuildDelay.String()) + appendFlags("delay", fmt.Sprintf("%d", testFlags.Delay)) + appendFlags("networkName", testFlags.NetworkName) + appendFlags("enableTele=", fmt.Sprintf("%v", testFlags.EnableTele)) + appendFlags("apiTimeout", fmt.Sprintf("%d", testFlags.ApiTimeout)) + appendFlags("mongoPassword", testFlags.MongoPassword) + appendFlags("coverageReportPath", testFlags.CoverageReportPath) + appendFlags("withCoverage", fmt.Sprintf("%v", testFlags.WithCoverage)) + appendFlags("proxyport", fmt.Sprintf("%d", testFlags.Proxyport))
+		keployAlias = keployAlias + addtionalFlags
```

Comment:
`appendFlags("withCoverage", fmt.Sprintf("%v", testFlags.WithCoverage))`

This should be
`appendFlags("withCoverage=", fmt.Sprintf("%v", testFlags.WithCoverage))
`

Reply:
done